db/seeds.rb

```diff
@@ -2,7 +2,14 @@
 MonthlyReport.destroy_all
 Tag.destroy_all
 
-user = FactoryGirl.create(:user, email: 'test@example.com', password: 'password')
-FactoryGirl.create(:user_role, :admin, user: user)
+if Rails.env.development?
+  user = FactoryGirl.create(:user, email: 'test@example.com', password: 'password')
+  FactoryGirl.create(:user_role, :admin, user: user)
 
-5.times { FactoryGirl.create(:monthly_report_tag) }
+  FactoryGirl.create(:user, email: 'testuser@example.com', password: 'password')
+
+  5.times { FactoryGirl.create(:monthly_report_tag) }
+else
+  user = User.create(name: 'admin', email: 'test@example.com', password: 'password', employee_code: 'x', entry_date: Date.today, beginner_flg: true)
```

Comment:
__[rubocop]__
* Line is too long. [148/128] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)